bridgev2/database/upgrades/00-latest.sql

```diff
@@ -128,7 +128,11 @@ CREATE TABLE disappearing_message (
 	timer        BIGINT NOT NULL,
 	disappear_at BIGINT,
 
-	PRIMARY KEY (bridge_id, mxid)
+	PRIMARY KEY (bridge_id, mxid),
+    CONSTRAINT fk_disappearing_message_portal
```

Comment:
Name should be `disappearing_message_portal_fkey` to match all the existing foreign keys